devTools/cardCreator/cards_to_import/Doom.spell.card.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "Doom",
+  "description": "The enemy General is DOOMED and will die after three turns",
+  "cardId": "Doom",
+  "spriteName": "icon_f4_doom",
+  "type": "SPELL",
+  "spells": [
+    {
+      "spellId": "doom",
+      "action": {
+        "enemyHitChanges": 0,
+        "apChange": 0,
+        "hpChange": -9999,
+        "mpChange": 0,
+        "poison": false,
+        "makeStun": false,
+        "makeDisarm": false,
+        "noDisarm": false,
+        "noPoison": false,
+        "noStun": false,
+        "noBadEffect": false,
+        "noAttackFromWeakerOnes": false,
+        "disableHolyBuff": false,
+        "addSpell": false,
+        "killsTarget": false,
```

Comment:
```suggestion
        "hpChange": 0,
        "mpChange": 0,
        "poison": false,
        "makeStun": false,
        "makeDisarm": false,
        "noDisarm": false,
        "noPoison": false,
        "noStun": false,
        "noBadEffect": false,
        "noAttackFromWeakerOnes": false,
        "disableHolyBuff": false,
        "addSpell": false,
        "killsTarget": true,
```

Reply:
If kills target works on heros, this is probably a better change.

Reply:
done